src/Pub/API/DTOs/ProjectDto.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace API.DTOs
+{
+    public class ProjectDto
```

Comment:
Since we will be using this ProjectDto to display a list of projects in the UI, we will need to add a few more properties to this including ProjectMember and Technologies. Since we don't have those yet it's okay if we don't add them now - instead i recommend we add a `TODO` comment so we don't forget.